Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift

```diff
@@ -74,7 +74,7 @@ final class LinearAlgebraTests: XCTestCase {
             }
             let aReconstituted = matmul(u!, matmul(s.diagonal(), 
                                         transposed: false, v!, transposed: true))
-            assertEqual(aReconstituted, a, accuracy: 1e-5)
+            assertEqual(aReconstituted, a, accuracy: 1e-3)
```

Comment:
> The accuracy of svd had to be changed for the tests to pass.

What error happens if you don't lower the accuracy? 1e-3 is a low accuracy value, which seems concerning - perhaps there's an implementation bug (mismatch between Swift and Python TensorFlow).

Reply:
@dan-zheng /mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:77: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("2.3218951") is not equal to ("2.3218844") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:77: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("-3.1908154") is not equal to ("-3.1908028") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:89: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("2.3218951") is not equal to ("2.3218844") +/- ("1e-05") -
/mnt/g/code/swift-apis/Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift:89: error: LinearAlgebraTests.testSVD : XCTAssertEqual failed: ("-3.1908154") is not equal to ("-3.1908028") +/- ("1e-05") -

The values seem very close to me. maybe i overdid it by decreasing it to 1e-3

Reply:
Thanks for sharing! Always using the tightest accuracy possible (`3e-5`) sounds great!